create_infotexts.py

```diff
@@ -8,88 +8,89 @@
 Outputs <Fotonummer>.info files in a subdirectory ./infofiles/
 """
 
-import os
 import json
 import re
 import pandas as pd
 import batchupload.helpers as helpers
 
 people_mapping_string = """{
```

Comment:
Done. https://phabricator.wikimedia.org/T159107

However, by breaking it out the logic for creating wikitable also needs to be adressed - or left as is but non-working.